rubocop/rubocop_rails.yml

```diff
@@ -3,11 +3,34 @@ inherit_from: rubocop.yml
 AllCops:
   Exclude:
     - db/schema.rb
+    - 'bin/**/*'
+    - 'config/**/*'
```

Comment:
If we change things in here, I'd rather see that they adhere to our coding standard; however, I know that some cops are violated out of the box, and also that some, like module length, may not make sense to adhere to in these files. However, turning off all cops seems a bit too generic for me.